internal/services/error_renderer.go

```diff
@@ -35,3 +46,33 @@ func renderNotFoundOrDAOError(w http.ResponseWriter, r *http.Request, err error,
 		renderError(w, r, payloads.NewDAOError(r.Context(), resource, err))
 	}
 }
+
+func writeEmptyResponse(w http.ResponseWriter, _ *http.Request, code int) {
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	w.Header().Set("Content-Length", "0")
+	w.WriteHeader(code)
+}
+
+func writeOk(w http.ResponseWriter, r *http.Request) {
+	writeEmptyResponse(w, r, http.StatusOK)
+}
+
+func writeNotFound(w http.ResponseWriter, r *http.Request) {
+	writeEmptyResponse(w, r, http.StatusNotFound)
+}
+
+func writeBadRequest(w http.ResponseWriter, r *http.Request) {
+	writeEmptyResponse(w, r, http.StatusBadRequest)
+}
+
+func writeServiceUnavailable(w http.ResponseWriter, r *http.Request) {
+	writeEmptyResponse(w, r, http.StatusServiceUnavailable)
+}
+
+func writeNoContent(w http.ResponseWriter, r *http.Request) {
+	writeEmptyResponse(w, r, http.StatusNoContent)
+}
+
+func writeUnauthorized(w http.ResponseWriter, r *http.Request) {
+	writeEmptyResponse(w, r, http.StatusUnauthorized)
+}
```

Comment:
I moved these function from various places into a single place and renamed them so it`s more readable.